app/src/main/java/cs465/illinois/edu/dogonthequad/CreateMeetupActivity.java

```diff
@@ -112,6 +112,7 @@ private void clearPreviousActivities() {
         finish();
     }
 
+
```

Comment:
Shouldn't `CreateMeetupSocialLevelActivity` be added to the activity list in this file?